test/integration/include/limit.test.js

```diff
@@ -0,0 +1,320 @@
+'use strict';
+
+const chai = require('chai'),
+  Sequelize = require('../../../index'),
+  expect = chai.expect,
+  Support = require(__dirname + '/../support'),
+  DataTypes = require(__dirname + '/../../../lib/data-types'),
+  Promise = Sequelize.Promise;
+
+describe(Support.getTestDialectTeaser('Include'), () => {
+  describe('LIMIT', () => {
+
+    let User = false;
```

Comment:
Please share these models using test scopes rather than global variables 

```js
beforeEach(function() {
  this.Model = sequelize.define(/***/)
})

it('your test', function() {
   this.Model.findAll(/***/)
}); 
```